idd/Energy+.idd.in

```diff
@@ -45619,7 +45615,131 @@ Coil:Heating:DX:VariableSpeed,
         \note quadratic curve = a + b*ffa + c*ffa**2
         \note cubic curve = a + b*ffa + c*ffa**2 + d*ffa**3
         \note ffa = Fraction of the full load Air Flow
+						
+COILSYSTEM:INTEGRATEDHEATPUMP:AIRSOURCE,
+	   \memo This object is used for air-source integrated heat pump, a collection of its working modes.
+  A1, \field Name
+       \required-field
+       \type alpha
+       \reference IntegratedHeatPumps
+       \note Unique name for this instance of an air-source integrated heat pump.
+  A2, \field Indoor Air Cooling Coil Inlet Node Name
```

Comment:
defining “cooling coil” “heating coil” and “outdoor” are necessary, as the ASIHP contains multiple nodes, separately for cooling coils, heating coils, and water heating coils with outdoor air node. ASIHP will pass these nodes to “child” coils accordingly. And defining separate “cooling coil” “heating coil” nodes is needed to accommodate the inputs of AirLoopHVAC:UnitaryHeatPump:AirToAir (a related object). We tried to shorten them, by deleting “indoor”, and add “Name” to all endings.